modules/core/04-channel/keeper/upgrade_test.go

```diff
@@ -1189,3 +1189,63 @@ func (suite *KeeperTestSuite) TestChanUpgradeCancel() {
 		})
 	}
 }
+
+func (suite *KeeperTestSuite) TestWriteChannelUpgradeAck() {
+	var (
+		path            *ibctesting.Path
+		proposedUpgrade types.Upgrade
+	)
+
+	testCases := []struct {
+		name                 string
+		malleate             func()
+		hasPacketCommitments bool
+	}{
+		{
+			"success with no packet commitments",
+			func() {},
+			false,
+		},
+		{
+			"success with packet commitments",
+			func() {
+				// manually set packet commitment
+				_, err := path.EndpointA.SendPacket(suite.chainB.GetTimeoutHeight(), 0, ibctesting.MockPacketData)
```

Comment:
We may as well check the sequence rather than discarding, right? No harm to do `suite.Require().Equal(uint64(1), sequence)`

Reply:
ah right, good point!

Reply:
similar occurrence of this in the msg_server_test.